### ISSUE TITLE
 On open new tab, the dropdown is not shown, take advantage of it?

### ISSUE THREAD

**Issue by Noitidart** [mozilla/universal-search#212]
Redirected from - https://github.com/bwinton/VerticalTabs/issues/157#issuecomment-218587859

When we first open a new tab, the dropdown that shows after we start typing in the URL bar, with hints,is not shown. Did any ideas come up on taking advantage of that space? Maybe show a small one?

I did a mockup/brainstorm here - https://github.com/Noitidart/AppLauncher

**Comment by jaredhirsch** [comment]
@Noitidart thanks for the suggestion! We actually discussed opening the popup when the urlbar is focused, and showing some 'top sites' kind of stuff--very similar to your mockup.

However, the scope of this project was dramatically reduced a couple months ago, and we are now _only_ experimenting with whether server recommendations will provide value, no interesting UI explorations. I actually have a deprecated [earlier version](https://github.com/mozilla/universal-search-addon) of this prototype that replaced the awesomebar contents with an iframed html5 page, but it was abandoned because of the change in product requirements.

Activity Stream is doing a lot of interesting stuff on the new tab page, and it's very likely that, if Activity Stream finds an audience, it'll lead to richer metadata being integrated into client-side storage, and we'll surface that data in the awesomebar.

Thanks for the suggestion and your enthusiasm! Closing this bug.
